instrumentation/lettuce-5.0/src/main/java/com/nr/fit/lettuce/instrumentation/NRHolder.java

```diff
@@ -0,0 +1,69 @@
+package com.nr.fit.lettuce.instrumentation;
+
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Segment;
+import com.newrelic.api.agent.Token;
+
+public class NRHolder
+{
```

Comment:
Formatting. This parenthesis usage is inconsistent with the rest of the codebase.